python/lsst/sims/GalSimInterface/galSimCatalogs.py

```diff
@@ -363,6 +363,9 @@ def get_fitsFiles(self):
         majorAxis = self.column_by_name('majorAxis')
         positionAngle = self.column_by_name('positionAngle')
         sindex = self.column_by_name('sindex')
+        gamma1 = self.column_by_name('shear1')
```

Comment:
The unit test was failing because, for databases of stars, shear1, shear2, and kappa are not defined.  I added default values of zero to the GalSimCatalog classes.

Should we rather name these columns gamma1, gamma2, and kappa so that they agree with the community standard?  Shear maps have not yet been implemented in fatboy so, though this might involve a change in sims_catUtils, it won't have any real effect.

Reply:
I originally had them labeled as gamma1, gamma2, and kappa, but noticed that the classes in galSimPhoSimCatalogs.py used shear1 and shear2, so I switched to be consistent with that.   I'm happy to go back to gamma1 and gamma2 for this new code.

Reply:
I think the 'shear1' 'shear2' convention is copied over from sims_catUtils, which may be copied over from an old PhoSim convention.  I think we should change it for the sake of transparency.  I will change it in sims_catUtils.